servicetalk-log4j2-mdc/src/main/java/io/servicetalk/log4j2/mdc/DefaultServiceTalkThreadContextMap.java

```diff
@@ -17,8 +17,41 @@
 
 import io.servicetalk.log4j2.mdc.utils.ServiceTalkThreadContextMap;
 
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
 /**
- * Empty subclass to differentiate uses of MDC.
+ * Subclass which uses the {@link io.servicetalk.concurrent.api.CapturedContext} mechanism for propagation.
  */
 public class DefaultServiceTalkThreadContextMap extends ServiceTalkThreadContextMap {
```

Comment:
Should we do anything with `ServiceTalkTracingThreadContextMap`?

Reply:
That one was unclear to me. We currently haven't upgraded the opentracing library to use the `CapturedContextProvider` APIs, and I _thought_ we didn't plan to. In my minds eye it is good to keep the propagation of the MDC and the tracing contexts in step, so I left it as AsyncContext for both.